tests/test_integration.py

```diff
@@ -77,7 +77,7 @@ def test_background_steps(docker_main_and_nonci):
 configs = Variations([dict(name="Bad step", command=["ls", "not_a_file"]),
                       dict(name="Good bg step", command=["touch", "file"],
                       background=True, artifacts="file")])
-""")
+""", force_installed=docker_main_and_nonci.nonci)
```

Comment:
You are passing the value you've got from the runner to the runner.
I guess, the runner can do that by itself.

Reply:
I'm not sure how exactly runner should do it itself. Do you suggest to always require `force_installed` in case of nonci?

Reply:
Probably. To answer I have to understand the intention of this change.
My comment is just observation.